modules/core/src/main/java/org/apache/synapse/rest/RESTUtils.java

```diff
@@ -153,4 +153,17 @@ private static void handleException(String msg, Throwable t) {
         throw new SynapseException(msg, t);
     }
 
+    /**
+     * Identify the API using the API context.
+     *
+     * @param path    request path
+     * @param context API context
+     */
+    public static boolean isRESTApiPath(String path, String context) {
```

Comment:
IMHO, the method name is not explanatory enough. Javadoc is also not clear. Shall we change that?

Reply:
Fixed